ipa-core/src/report/hybrid_info.rs

```diff
@@ -0,0 +1,65 @@
+use crate::report::{hybrid::NonAsciiStringError, KeyIdentifier};
+
+const DOMAIN: &str = "private-attribution";
+
+#[derive(Debug)]
```

Comment:
Why only this one has `Debug`?

Reply:
because that's the only stuct here that is used elsewhere. It didn't make sense to implement Debug for dead code. The followup PR will change this when it uses these types.